README.md

```diff
@@ -69,7 +69,7 @@ Translations
 ------------
 
 Changes to translations as well as new translations can be submitted to
-[Dash Core's Transifex page](https://www.transifex.com/projects/p/dash/).
+[Dash Core's Transifex page](https://www.transifex.com/bitcoin/dash/).
```

Comment:
15514 bad link This should maybe be DNM

Reply:
i just realized i broke the link on my end, the original was fine, should we still drop?

Reply:
I don't really see the value of it but :shrug: sure for now?